tad/migrations/versions/b62dbd9468e4_create_status_user_and_task_table.py

```diff
@@ -60,10 +60,10 @@ def upgrade() -> None:
     op.bulk_insert(
         status,
         [
-            {"name": "Todo", "sort_order": 1},
-            {"name": "In Progress", "sort_order": 2},
-            {"name": "Review", "sort_order": 3},
-            {"name": "Done", "sort_order": 4},
+            {"name": "todo", "sort_order": 1},
```

Comment:
Its not a good idea to edit the migrations. Some may already applied them.

Reply:
To add to this: if you want to change the content of a db, the way to go is to add a _new_ migration that applies the above.